### PR TITLE
feat: using powershell to extract versions

### PR DESCRIPTION
**Pull request by adrukh**
feat: Using powershell on windows machines to extract version from .dll files.
On .csproj manifests with "custom" configuration, reference items may lead to folders out of the project's scope, or to folders that do not imply on the package name and version. Until this change the plugin ignored these references. Using powershell the plugin can extract the version from the .dll file directly. This is enabled on windows machines only and only in case of .csproj manifest.
Most projects that follows best-practices uses either project.assets.json manifest or packages.config.
Though this is not a common use-case, now it can be fully supported.

### CODE REVIEW

File: lib/dependency.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 var debug = require('debug')('snyk');
+var childProcess = require('child_process');
```
Comment (orkamara) [inline lib/dependency.js]: - [ ] test

**Comment by dkontorovskyy** [comment]
@adrukh I think we can close this. Right?

**Comment by adrukh** [comment]
Indeed! 2 birthdays are more than enough for a PR :)

**Comment by dkontorovskyy** [comment]
@adrukh always thought that 3 is max :)

**Pull request closed without merging**